tests/impl/Lua-5.1-tests/CMakeLists.txt

```diff
@@ -21,12 +21,22 @@ set(SUITE_TEST_LIB_TARGETS "")
 set(SUITE_TEST_LIB_OUT_DIR ${CMAKE_CURRENT_BINARY_DIR})
 
 foreach(SUITE_TEST_LIB_NAME ${SUITE_TEST_LIB_NAMES})
-  # e.g. target "lua_51_test_lib1", compiled from "lib1.c", produces "lib1.so"
+  # e.g. target "test_lua_51_lib1", compiled from "lib1.c", produces "lib1.so"
+  set(TEST_LIB_PATH ${SUITE_DIR}/suite/libs)
   set(SUITE_TEST_LIB_TARGET "test_lua_51_${SUITE_TEST_LIB_NAME}")
-  set(SUITE_TEST_LIB_SOURCE ${SUITE_DIR}/suite/libs/${SUITE_TEST_LIB_NAME}.c)
+  set(SUITE_TEST_LIB_SOURCE "")
+  list(APPEND SUITE_TEST_LIB_SOURCE ${TEST_LIB_PATH}/${SUITE_TEST_LIB_NAME}.c)
```

Comment:
No need for doing that in two lines. If you do 

```cmake
set(SUITE_TEST_LIB_SOURCE ${TEST_LIB_PATH}/${SUITE_TEST_LIB_NAME}.c)
```

it'll be exactly the same. (also I'd name it SUITE_TEST_LIB_SOURCES now)

Reply:
Fixed.